apps/client/src/app/[user]/components/RecommendSection/index.tsx

```diff
@@ -1,23 +1,83 @@
+"use client";
+
+import SearchStudyInput from "@/app/[user]/components/SearchStudyInput";
+import { useSearchStudyQueryObject } from "@/app/api/groups/query";
+import imgEmpty from "@/asset/img/img_empty.png";
+import { useQuery } from "@tanstack/react-query";
+import Image from "next/image";
+import { useSearchParams } from "next/navigation";
 import RecommendCard from "./RecommendCard";
 import {
+  emptyGuideStyle,
+  emptyWrapper,
+  recommendHeaderContentWrapper,
   recommendHeaderWrapper,
   recommendSectionWrapper,
   recommendStudyTitle,
+  searchedStudyCountStyle,
+  studyListWrapper,
 } from "./index.css";
 
-const RecommendStudySection = () => (
-  <section
-    className={recommendSectionWrapper}
-    aria-labelledby="recommend-title"
-  >
-    <div className={recommendHeaderWrapper}>
-      <h2 id="recommend-title" className={recommendStudyTitle}>
-        추천 스터디
-      </h2>
-      <input />
-    </div>
-    <RecommendCard />
-  </section>
-);
+const RecommendStudySection = () => {
+  const searchParam = useSearchParams();
+  const searchPattern = searchParam.get("search");
+
+  const { data: studyList } = useQuery(
+    useSearchStudyQueryObject({ searchPattern: searchPattern || "" }),
+  );
```

Comment:
```suggestion
  const searchPattern = searchParam.get("search")  || "";

  const { data: studyList } = useQuery(
    useSearchStudyQueryObject({ searchPattern }),
  );
```

사소하지만 이게 더 괜찮은 것 같죠?